src/scripts/update_funding_round.js

```diff
@@ -111,26 +111,63 @@ const main = async () => {
 
             let allProposals = await getProposalsSelectQuery(`{Round} = ${curRoundNumber}`)
             const tokenPrice = await getTokenPrice()
-            const maxGrantUSD = curRound.get('Max Grant USD')
-            const earmarkedUSD = curRound.get('Earmarked USD')
-            const fundingAvailableUSD = curRound.get('Funding Available USD')
-
-            const maxGrantOCEAN = maxGrantUSD / tokenPrice
-            const earmarkedOCEAN = earmarkedUSD / tokenPrice
-            const fundingAvailableOCEAN = fundingAvailableUSD / tokenPrice
-
-            // Enter Due Diligence period
-            const roundUpdate = [{
+            const USDPrice = 1/tokenPrice
+            const basisCurrency = curRound.get('Basis Currency')
+
+            let maxGrant = 0
+            let earmarked = 0
+            let fundingAvailable = 0
+
+            let maxGrantUSD = 0
+            let earmarkedUSD = 0
+            let fundingAvailableUSD = 0
+
+            switch(basisCurrency) {
+                case 'USD' :
+                maxGrantUSD = curRound.get('Max Grant USD')
+                earmarkedUSD = curRound.get('Earmarked USD')
+                fundingAvailableUSD = curRound.get('Funding Available USD')
+
+                maxGrant = maxGrantUSD/ tokenPrice
+                earmarked = earmarkedUSD / tokenPrice
+                fundingAvailable = fundingAvailableUSD / tokenPrice
+                break
+
+                case 'OCEAN': 
+                const maxGrantOCEAN = curRound.get('Max Grant')
+                const earmarkedOCEAN = curRound.get('Earmarked')
+                const fundingAvailableOCEAN = curRound.get('Funding Available')
+
+                maxGrant = maxGrantOCEAN
+                earmarked = earmarkedOCEAN
+                fundingAvailable = fundingAvailableOCEAN 
+
+                maxGrantUSD = maxGrantOCEAN / USDPrice
+                earmarkedUSD = earmarkedOCEAN / USDPrice
+                fundingAvailableUSD = fundingAvailableOCEAN / USDPrice               
```

Comment:
This doesn't look right for me, I think it should be `maxGrantOCEAN * oceanTokenPrice`

Reply:
`maxGrantOcean * oceanTokenPrice` is equivalent to `maxGrantOcean / USDPrice`, which represents the value of a USD in Ocean, aka `1 / tokenPrice`

Reply:
right, that USDPrice name got me confused. All good then.